ui/components/app/snaps/copyable/copyable.js

```diff
@@ -1,26 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Box from '../../../ui/box';
-import Typography from '../../../ui/typography';
+
 import {
   AlignItems,
   BorderRadius,
+  Display,
   JustifyContent,
-  OVERFLOW_WRAP,
-  FLEX_DIRECTION,
-  TypographyVariant,
+  OverflowWrap,
+  FlexDirection,
+  TextVariant,
   BackgroundColor,
   TextColor,
   Color,
 } from '../../../../helpers/constants/design-system';
 import { useCopyToClipboard } from '../../../../hooks/useCopyToClipboard';
-import { Icon, IconName, IconSize } from '../../../component-library';
+import {
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+  Box,
+} from '../../../component-library';
 
 export const Copyable = ({ text }) => {
   const [copied, handleCopy] = useCopyToClipboard();
   return (
     <Box
       className="copyable"
+      display={Display.Flex}
```

Comment:
before
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/6f1b002b-b9b1-4fdf-a2f3-9a9d2752f227)
after
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/a7f93d3e-b9ee-4a63-b46e-8f3645c106df)

Reply:
Made some updates so it uses the `ButtonIcon` component

### After


https://github.com/MetaMask/metamask-extension/assets/8112138/0aecd1e7-70b4-4418-8e69-b67f7792b740